api/v1alpha1/clusterextension_types.go

```diff
@@ -386,28 +393,30 @@ type PreflightConfig struct {
 	// consequences of upgrading a CRD, such as data loss.
 	//
 	// This field is required if the spec.install.preflight field is specified.
+	//+kubebuilder:validation:Required
 	CRDUpgradeSafety *CRDUpgradeSafetyPreflightConfig `json:"crdUpgradeSafety"`
 }
 
 // CRDUpgradeSafetyPreflightConfig is the configuration for CRD upgrade safety preflight check.
 type CRDUpgradeSafetyPreflightConfig struct {
-	// policy is used to configure the state of the CRD Upgrade Safety pre-flight check.
+	// enforcement is used to configure the state of the CRD Upgrade Safety pre-flight check.
 	//
 	// This field is required when the spec.install.preflight.crdUpgradeSafety field is
 	// specified.
```

Comment:
I think I saw a comment from Joel about not needing the conditional statement here. We can just say "enforcement is required" unconditionally (it is implicit that it being required is conditional on the parent struct being specified).

Reply:
[[1]](https://github.com/openshift/api/pull/2067#discussion_r1812368363) comment that mentioning optional/required isn't necessary on status

Happy to make the change, but I didn't see a comment like this (maybe yet!).